presidio-anonymizer/presidio_anonymizer/anonymizers/anonymizer.py

```diff
@@ -22,3 +24,13 @@ def validate(self, params: dict = None) -> None:
     def anonymizer_name(self) -> str:
         """Return anonymizer name."""
         pass
+
+    @staticmethod
+    def get_anonymizers():
+        """Return all anonymizers classes currently available."""
+        if not Anonymizer._anonymizers:
+            Anonymizer._anonymizers = {
+                cls.anonymizer_name(cls): cls
```

Comment:
In pycharm, this is complaining that it "Expected type 'Anonymizer' but got 'Type[Anonymizer]' instead" which seems to make sense... is this working as it should?